app/lib/submission_builder/document.rb

```diff
@@ -63,5 +63,16 @@ def add_non_zero_claimed_value(xml, elem_name, claimed)
         xml.send(elem_name, claimed: claimed_value)
       end
     end
+
+    def add_positive_claimed_value_or_zero(xml, elem_name, claimed)
+      claimed_value = calculated_fields.fetch(claimed)
+      if claimed_value.present?
+        if claimed_value.to_i < 0
+          xml.send(elem_name, claimed: 0)
+        else
+          xml.send(elem_name, claimed: claimed_value)
+        end
```

Comment:
I think you could simplify this a bit
```suggestion
        xml.send(elem_name, claimed: Math.max(claimed_value, 0))
```